src/configs/python.rs

```diff
@@ -21,7 +21,7 @@ impl<'a> RootModuleConfig<'a> for PythonConfig<'a> {
             pyenv_prefix: "pyenv ",
             python_binary: VecOr(vec!["python", "python3", "python2"]),
             scan_for_pyfiles: true,
-            format: "via [${symbol}${pyenv_prefix}${version}( \\($virtualenv\\))]($style) ",
+            format: "via [${symbol}${pyenv_prefix}${version} (\\($virtualenv\\))]($style)",
```

Comment:
I think this should be 
```suggestion
            format: "via [${symbol}${pyenv_prefix}(${version} )(\\($virtualenv\\))]($style)",
```